src/components/ClockFace.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect, createContext } from "react";
+import classnames from "classnames";
+import SettingsModal from "./SettingsModal";
+import "./ClockFace.scss";
+
+export const ModalContext = createContext();
+
+const ClockFace = () => {
+  const [sessions, setSessions] = useState([1500, 300, 1500, 300]);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isActive, setIsActive] = useState(false);
+  const [currentSessionIndex, setCurrentSessionIndex] = useState(0);
+  const [currentSeconds, setCurrentSeconds] = useState(sessions[0]);
+
+  const settingsModalValues = {
+    isModalOpen, setIsModalOpen, sessions, setSessions
+  }
```

Comment:
I've considered using the useMemo hook to boost performance, but I think this is quite small, especially sessions can only be changed by the modal only anyways.

Reply:
You're right. You don't really need to worry about memo-ing your code unless you encounter performance problem.